metrics/config.go

```diff
@@ -144,13 +144,19 @@ func NewStackdriverClientConfigFromMap(config map[string]string) *StackdriverCli
 	}
 }
 
-// Record applies the `ros` Options to `ms` and then records the resulting
+// RecordN applies the `ros` Options to each measurement in `mss` and then records the resulting
```

Comment:
Could we call this RecordBatch? RecordN makes me think I am recording the same measurement N times instead of N different measurements.